cypress/support/commands.js

```diff
@@ -450,3 +469,105 @@ Cypress.Commands.add('smiteUser', (amountToSmite, isMotion) => {
+    ? 'Test motion annotation'
+    : 'Test annotation';
+  const colonyName = Cypress.config().colony.name;
+
+  cy.login();
+  cy.visit(`/colony/${colonyName}`);
+  cy.changeColonyname(colonyName, newName);
+
+  const filePath = 'cypress/fixtures/images/jaya-the-beast.png';
+  cy.get('input[type="file"]').selectFile(filePath, { force: true });
+
+  cy.get('textarea').click().type(annotationText);
+  cy.getBySel('confirmButton').click();
+
+  cy.getBySel('actionHeading', { timeout: 60000 }).should(
+    'have.text',
+    isMotion ? 'Change colony details' : 'Colony details changed',
+  );
+  cy.checkUrlAfterAction(colonyName);
+});
+
+Cypress.Commands.add(
+  'updateTokens',
+  (updatedColonyName, tokenProviderColonyName, isMotion) => {
+    cy.getColonyTokenAddress(tokenProviderColonyName);
+
+    cy.visit(`/colony/${updatedColonyName}`);
+
+    cy.getBySel('newActionButton', { timeout: 90000 }).click();
+    cy.getBySel('fundsDialogIndexItem').click();
+    cy.getBySel('manageTokensDialogItem').click();
+
+    cy.get('@existingTokenAddress').then((address) => {
+      cy.get('input').last().click().type(address);
+    });
+    cy.getBySel('confirm').click();
+
+    cy.getBySel('actionHeading', { timeout: 60000 }).should(
+      'have.text',
+      isMotion ? 'Change colony details' : `Colony details changed`,
+    );
+
+    cy.checkUrlAfterAction(updatedColonyName);
+  },
+);
+
+Cypress.Commands.add('unlockToken', (colony) => {
+  cy.getBySel('newActionButton', { timeout: 60000 }).click();
+  cy.getBySel('fundsDialogIndexItem').click();
+  cy.getBySel('unlockTokenDialogIndexItem').click();
+
+  cy.getBySel('unlockTokenConfirmButton').click();
+
+  cy.getBySel('actionHeading', { timeout: 100000 }).should(
+    'include.text',
+    `Unlock native token ${colony.nativeToken}`,
+  );
+
+  cy.url().should(
+    'contains',
+    `${Cypress.config().baseUrl}/colony/${colony.name}/tx/0x`,
+  );
+});
+
+Cypress.Commands.add('managePermissions', (colonyName, isMotion) => {
+  const annotationText = 'I am giving you the power to do things';
+
+  cy.login();
+
+  cy.visit(`/colony/${colonyName}`);
+
+  cy.getBySel('newActionButton', { timeout: 60000 }).click();
+  cy.getBySel('advancedDialogIndexItem').click();
+  cy.getBySel('managePermissionsDialogIndexItem').click();
+
+  if (!isMotion) {
+    cy.getBySel('permissionUserSelector').click({ force: true });
+    cy.getBySel('permissionUserSelectorItem').last().click();
+  }
+  cy.getBySel('permission').eq(1).click({ force: true });
+  cy.getBySel('permission').eq(2).click({ force: true });
+  cy.getBySel('permission').eq(3).click({ force: true });
+  cy.getBySel('permissionAnnotation').click().type(annotationText);
+
+  cy.getBySel('permissionConfirmButton').click();
+
+  cy.getBySel('actionHeading', { timeout: 100000 }).contains(
+    isMotion
+      ? 'Remove the administration, architecture, funding permissions'
+      : `Assign the administration, funding, architecture permissions in Root to`,
```

Comment:
Would it be better to pull these from the language files?

Reply:
Yeah, I think we shouldn't really make the tests even more compliated